src/applications/vre/28-8832/config/chapters/veteran-information/staticVeteranInformation.js

```diff
@@ -0,0 +1,13 @@
+import VeteranInformationComponent from './staticVeteranComponent.jsx';
+
+export const schema = {
+  type: 'object',
+  properties: {},
+};
+
+export const uiSchema = {
+  'ui:description': VeteranInformationComponent,
+  'ui:options': {
+    hideOnReview: true,
```

Comment:
I think eventually static veteran information needs to be displayed on the review page of the form